### ISSUE TITLE
Incorrect filename, fileName property when file path contains space 

### ISSUE THREAD

**Issue by modethirteen** [stacktracejs/error-stack-parser#35]
In Node.JS (v7+), the `fileName` and `functionName` properties result in incorrect values when parsing a callsite that has spaces in the filename.

## Error Object
```
Error: The file could not be uploaded to the element `#file-image` because it does not exist on the page
    at setFavicon (/Users/andyv/MindTouch/QA-Automated-Testsuite/tests_active/testrun/Control Panel/Branding/customSiteFavicon.js:18:24)
```

## Expected Behavior
```json
[
  {
    "columnNumber": 24,
    "lineNumber": 18,
    "fileName": "/Users/andyv/MindTouch/QA-Automated-Testsuite/tests_active/testrun/Control Panel/Branding/customSiteFavicon.js",
    "functionName": "setFavicon",
    "source": "    at setFavicon (/Users/andyv/MindTouch/QA-Automated-Testsuite/tests_active/testrun/Control Panel/Branding/customSiteFavicon.js:18:24)"
  }
]
```

## Current Behavior
```
[
  {
    "columnNumber": 24,
    "lineNumber": 18,
    "fileName": "Panel/Branding/customSiteFavicon.js",
    "functionName": "setFavicon (/Users/andyv/MindTouch/QA-Automated-Testsuite/tests_active/testrun/Control",
    "source": "    at setFavicon (/Users/andyv/MindTouch/QA-Automated-Testsuite/tests_active/testrun/Control Panel/Branding/customSiteFavicon.js:18:24)"
  }
]
```

**Comment by niftylettuce** [comment]
This can be closed per #46 being resolved

**Comment by modethirteen** [comment]
👍
